tests/acceptance/10_files/09_insert_lines/nothing_to_repeat_error.cf

```diff
@@ -16,7 +16,7 @@ body common control
 bundle agent check
 {
   methods:
-      "" usebundle => dcs_passif_output("",
+      "" usebundle => dcs_passif_output(".*info: Inserted the promised line.*",
```

Comment:
Alright, so now there is a new info level log that is emitted when a line is inserted. Previously, there was no specific output expected, only output that would cause the test to fail (`nothing to repeat`).

Makes sense.